package com.marcosperboni.orderservice.dto;

import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Positive;

import java.math.BigDecimal;
import java.util.UUID;

public record CreateOrderItemRequest(
		@NotNull UUID productId,
		@NotNull @Positive Integer quantity,
		@NotNull @Positive BigDecimal unitPrice) {
}
